src/sys/wait.rs

```diff
@@ -1,58 +1,105 @@
-use libc::{self, c_int};
+//! This module contains the `wait()` and `waitpid()` functions, which are used to wait on and
```

Comment:
Short descriptive first line, then a blank line, then more description. Generate the docs locally with `rust doc --no-deps --open` to see what I'm talking about.